src/Compilers/VisualBasic/Portable/Lowering/LambdaRewriter/LambdaRewriter.vb

```diff
@@ -349,12 +362,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' <returns>A bound node that computes the pointer to the required frame</returns>
         Friend Overrides Function FramePointer(syntax As VisualBasicSyntaxNode, frameClass As NamedTypeSymbol) As BoundExpression
             Debug.Assert(frameClass.IsDefinition)
-            If currentFrameThis IsNot Nothing AndAlso currentFrameThis.Type = frameClass Then
+            If _currentFrameThis IsNot Nothing AndAlso _currentFrameThis.Type = frameClass Then
                 Return New BoundMeReference(syntax, frameClass)
             End If
 
             ' Otherwise we need to return the value from a frame pointer local variable...
-            Dim result As Symbol = framePointers(frameClass)
+            Dim result As Symbol = _framePointers(frameClass)
             Dim proxyField As FieldSymbol = Nothing
```

Comment:
It is to avoid warning saying "passed by reference before it has been assigned a value" when calling TryGetValue(result, proxyField).